pages/auction/index.js

```diff
@@ -15,9 +15,9 @@ const Auctions = () => {
       </Head>
       <NavBar personData={personData} />
       <Link href="/auction/new">
-        <a>
-          <h2 className={styles.auctionPageNavigation}>Create New Auction</h2>
-        </a>
+        <div className={styles.newAuctionBtnContainer}>
+          <button className={styles.newAuctionBtn}>Create New Auction</button>
```

Comment:
type attribute missing

Reply:
Whenever i'm trying to make any changes, all these errors which were coming yesterday, it's coming again.
![image](https://user-images.githubusercontent.com/86421817/132127405-aae20a5a-c56b-45b9-a296-f64d979a12b2.png)

Reply:
@Aditi-0725 I see you've prettier enabled, you could click on the Prettier (bottom right corner -- blue bar) and select ESLint. I personally don't use Prettier at all. You may uninstall it maybe? ESLint already does what prettier does plus a lot more things which prettier doesn't. 

Also this is something that might help you: https://www.digitalocean.com/community/tutorials/workflow-auto-eslinting

> If you have the ESLint extension installed you can use CTRL+SHIFT+P to open the Command Palette. Then search for ESLint: Fix all auto-fixable Problems and press ENTER (or RETURN).

This is one time though, I personally prefer auto-linting on file save (CTRL + S), it's mentioned in point 4.

Reply:
Okay, i've made the changes.